### PR TITLE
Auto refresh of Appointments

### PR DESCRIPTION
**Pull request by rajur7**
[JIRA link for Auto Refresh Story](https://bahmni.atlassian.net/browse/BAH-624)

Configuration:- 
    We need to add a configuration to enable this feature. We have to configure with a time interval by providing value to `autoRefreshIntervalInSeconds` in ***openmrs/apps/appointments/app.json*** in config.

### CODE REVIEW

File: ui/app/appointments/controllers/manage/list/appointmentsListViewController.js [hunk 1]
```diff
@@ -36,6 +38,53 @@ angular.module('bahmni.appointments')
                 $scope.isFilterOpen = $stateParams.isFilterOpen;
             };
 
+            var autoRefreshStatus = true;
```
Comment (sumanmaity112) [inline ui/app/appointments/controllers/manage/list/appointmentsListViewController.js]: @rajur7, Please move this ``var autoRefreshStatus = true;`` to top as this is the default value

File: ui/test/unit/appointments/controllers/manage/calendar/appointmentsCalendarViewController.spec.js [hunk 2]
```diff
@@ -474,5 +480,65 @@ describe('AppointmentsCalendarViewController', function () {
...
+
+    it('should call interval function when autoRefreshIntervalInSeconds is defined', function () {
+        appDescriptor.getConfigValue.and.callFake(function (value) {
+            if (value === 'autoRefreshIntervalInSeconds') {
+                return 10;
+            }
+            return undefined;
+        });
+
+        createController();
```
Comment (sumanmaity112) [inline ui/test/unit/appointments/controllers/manage/calendar/appointmentsCalendarViewController.spec.js]: @rajur7, If I am not wrong, you can pull `createController()` to `beforeEach` method

Comment (sowmika148) [reply ui/test/unit/appointments/controllers/manage/calendar/appointmentsCalendarViewController.spec.js]: @sumanmaity112 , we are calling createController method to initialise the private variables required for these tests.

Comment (sumanmaity112) [reply ui/test/unit/appointments/controllers/manage/calendar/appointmentsCalendarViewController.spec.js]: @sowmika148, then would it be `beforeEach` thing?

**Pull request closed without merging**